src/map/handler/Map.Drag.js

```diff
@@ -96,12 +96,12 @@ export var Drag = Handler.extend({
 	},
 
 	_onDown: function () {
```

Comment:
I'm thinking "fix the bug now, revisit later if users complain about the behaviour" :-)